src/generators/daf_file_generator.py

```diff
@@ -1,7 +1,7 @@
-import sys
+# import sys
```

Comment:
Can you delete all the parts that you are commenting out if they are not required?